core-web/libs/block-editor/src/lib/extensions/ai-image-prompt/ai-image-prompt.component.ts

```diff
@@ -36,10 +36,10 @@ import { AiImagePromptInputComponent } from './components/ai-image-prompt-input/
     changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class AIImagePromptComponent {
-    protected readonly vm$: Observable<VmAiImagePrompt> = inject(DotAiImagePromptStore).vm$;
+    protected readonly vm$: Observable<VmAiImagePrompt> = this.store.vm$;
 
     protected readonly ComponentStatus = ComponentStatus;
-    private store = inject(DotAiImagePromptStore);
+    constructor(private store: DotAiImagePromptStore) {}
```

Comment:
Why change inject to construct here?

Reply:
oh, yes, I forgot rollback. Testing something I forgot to fix it. Thanks.